src/Missings.jl

```diff
@@ -207,4 +208,256 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+
+"""
+   skipmissings(args...)
+
+Returns a tuple of iterators iterating over indices of elements `args`
+where all elements of `args` are not missing.
+
+# Examples
+```
+julia> x = [1, 2, missing, 4]; y = [1, 2, 3, missing];
+
+julia> tx, ty = skipmissings(x, y)
+(Missings.SkipMissings{Array{Union{Missing, Int64},1}}(Union{Missing, Int64}[1, 2, missing, 
+4], Base.Iterators.Zip{Tuple{Array{Union{Missing, Int64},1},Array{Union{Missing, Int64},1}}}
+((Union{Missing, Int64}[1, 2, missing, 4], Union{Missing, Int64}[1, 2, 3, missing])), 1), 
+Missings.SkipMissings{Array{Union{Missing, Int64},1}}(Union{Missing, Int64}[1, 2, 3, 
+missing], Base.Iterators.Zip{Tuple{Array{Union{Missing, Int64},1},Array{Union{Missing,
+ Int64},1}}}((Union{Missing, Int64}[1, 2, missing, 4], Union{Missing, Int64}[1, 2, 3, 
+ missing])), 2))
+
+julia> collect(tx)
+2-element Array{Any,1}:
+ 1
+ 2
+
+```
+"""
+function skipmissings(args...)
+    if args isa Tuple{Vararg{AbstractArray}}
+        if !mapreduce(length, isequal, args)
+            throw(ArgumentError("All arguments must have the same length"))
+        end
+
+        if !mapreduce(eachindex, isequal, args)
+            throw(ArgumentError("All arguments must have the same indices"))
+        end
+    end
+
+    ntuple(length(args)) do i
+        s = setdiff(1:length(args), i)
+        SkipMissings(args[i], args[s])
+    end
+end
+
+struct SkipMissings{V, T}
+    x::V
+    others::T
+end
+
+Base.@propagate_inbounds @inline function _anymissingindex(others::Tuple{Vararg{AbstractArray}}, i)    
```

Comment:
I think `@inline` is redundant.
```suggestion
Base.@propagate_inbounds function _anymissingindex(others::Tuple{Vararg{AbstractArray}}, i)    
```

Reply:
I'm not sure what happened but this suggestion got reverted. Any idea why? Hopefully it's the only one.